packages/client/src/app/components/modals/inventory/ItemGrid.tsx

```
@@ -117,7 +121,11 @@ export const ItemGrid = (props: Props) => {
     const options = getItemActions(item, inv.balance);
 
     return (
-      <TextTooltip key={item.index} text={[item.name, '', item.description ?? '']}>
+      <TextTooltip
+        key={item.index}
+        text={item.index ? [<ItemGridTooltip item={item} utils={utils} />] : []}
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)